MaxText/layers/linears.py

```diff
@@ -493,7 +493,7 @@ def load_balance_loss(self, top_k_indices, logits):
     loss = jnp.mean(density * density_prob) * (self.num_experts**2) * self.config.load_balance_loss_weight
     return loss
 
-  def get_einsum(self, rhs_mesh_axes: Tuple[Optional[str], ...] = ()):
+  def get_einsum(self, rhs_mesh_axes: Tuple[Optional[str], ...] = (),):
```

Comment:
Is this comma on purpose? similar comments for another line.

Reply:
fixed